modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogService.java

```diff
@@ -36,7 +36,7 @@
  * <p>TBD: events
  */
 public interface CatalogService {
-    String PUBLIC = "PUBLIC";
```

Comment:
was it really necessary to rename this constant?

Reply:
I would say that this remove is justified, but in another PR maybe. Name "PUBLIC" says nothing about what it is, and there's no javadoc to clarify things either.